src/openms_gui/source/VISUAL/TOPPASScene.cpp

```diff
@@ -467,13 +467,15 @@ namespace OpenMS
     }
 
     TOPPASEdge* edge;
+    // cppcheck-suppress uninitvar
```

Comment:
would `TOPPASEdge* edge(0)` also work? (less code and no dependency on cppcheck syntax)

Reply:
or TOPPASEdge* edge = nullptr;